### ISSUE TITLE
Doing something clearly wrong...

### ISSUE THREAD

**Issue by Majal-1036** [ChiWeiHsiao/DeepVO_PyTorch#57]
can you help me debug my code, when i run it my training loss comes around 28 for some reason , I downloaded the same repo and set the paths and everything but my training loss comes out to around 28, my img_mean is also different for some reason when I use the preprocessing.py to calculate the mean and std my values are around 80 and not between 0 and 1 although I am using the image_3 data set from KITTI like described in the repo. can someone guide in this regard ? can @ChiWeiHsiao or @alexart13 help here ?
I will try downloading the repo again if I made any mistakes, do i have to normalize the images ? even though the I believe the code normalizes them ?

**Comment by 110sha** [comment]
Have you solved this problem? I also encountered a similar problem.

**Comment by Majal-1036** [comment]
Unfortunately No, I just moved to a different Neural Network.
